R/data_analysis_page.R

```diff
@@ -0,0 +1,2146 @@
+                      "Number of trials indicated by node size and line thickness" = "networkp2"
+                    ),
+                    selected = "networkp2",
+                    width='100%'
+                  ),
+                  column(
+                    width = 6,
+                    radioButtons(
+                      inputId = ns('format_freq1'),
+                      label = 'Document format',
+                      choices = c('PDF', 'PNG'),
+                      inline = TRUE
+                    )
+                  ),
+                  column(
+                    width = 6,
+                    numericInput(inputId = ns('label_all'), label = 'Label size multiplier', value = 1.25)
+                  ),
+                  downloadButton(outputId = ns('downloadNetwork')),
+                  br(),
+                  br(),
+                  br(),
+                  verbatimTextOutput(outputId = ns("netconnect"))
+                ),
+                column(
+                  width = 6,
+                  plotOutput(outputId = ns("netGraphUpdating")),
+                  conditionalPanel(
+                    condition = "input.networkstyle_sub == 'networkp1'",
+                    ns = ns,
+                    p("Numbers on the line indicate the number of trials conducted for the comparison. The shaded areas indicate there exist multi-arm trials between the comparisons.")
+                  ),
+                  conditionalPanel(
+                    condition = "input.networkstyle_sub == 'networkp2'",
+                    ns = ns,
+                    p("The size of the nodes and thickness of edges represent the number of studies that examined a treatment and compared two given treatments respectively.")
+                  ),
+                  radioButtons(
+                    inputId = ns("networkstyle_sub"),
+                    label = "Please choose a network plot style",
+                    choices = c(
+                      "Number of trials shown on the line" = "networkp1",
+                      "Number of trials indicated by node size and line thickness" = "networkp2"
+                    ),
+                    selected = "networkp2",
+                    width = '100%'
+                  ),
+                  column(
+                    width = 6,
+                    radioButtons(
+                      inputId = ns('format_freq2'),
+                      label = 'Document format',
+                      choices = c('PDF', 'PNG'),
+                      inline = TRUE
+                    )
+                  ),
+                  column(
+                    width = 6,
+                    numericInput( inputId = ns('label_excluded'), label = 'Label size multiplier', value = 1.25)
+                  ),
+                  downloadButton(outputId = ns('downloadNetworkUpdate')),
+                  br(),
+                  br(),
+                  br(),
+                  verbatimTextOutput("netconnect_sub")
```

Comment:
This text output isn't showing (think I'm in the correct place - tab 1c under the download button for the sensitivity analysis)
<img width="1037" alt="Screenshot 2023-10-31 at 11 21 20" src="https://github.com/CRSU-Apps/MetaInsight/assets/98603626/89a3dedd-7a30-49a8-b91e-73890b1bf8e4">

Reply:
Well spotted. I've added the namespacing to that one now.